frontend/dockerfile/builder/build.go

```diff
@@ -71,6 +71,10 @@ func Build(ctx context.Context, c client.Client) (_ *client.Result, err error) {
 		return nil, capsError
 	}
 
+	dockerIgnorePatterns, err := bc.DockerIgnorePatterns(ctx)
```

Comment:
I don't think this needs to be resolved here and written to `ConvertOpt`. `dockerui.Client` is already passed, so it can just call this method when needed.